pipes/using_pipes.md

```diff
@@ -1,20 +1,22 @@
 # Using Pipes
 
-Like a filter, a pipe also takes data as input and transforms it to the desired output. A basic example of using pipes is shown below:
+Like a filter, a pipe also takes data as input and transforms it into the desired output. To use a pipe, you can use the pipe operator ( | ) followed by the name of the pipe function inside the interpolation expression. All pipes work this way. A basic example of using pipes is shown below:
 
 ```javascript
 import { Component } from '@angular/core';
 
 @Component({
   selector: 'product-price',
-  template: `<p>Total price of product is {{ price | currency }}</p>`
+  template: `<p>Total price of product is {{ price | currency }}</p>`,
 })
 export class ProductPrice {
   price = 100.1234;
 }
 ```
 
-[View Example](http://plnkr.co/edit/JIdYGCz1U9ElEpCArg01?p=preview)
+[View Example](https://stackblitz.com/github/rangle/angular-book-examples/tree/feat-using-pipes)
+
+Angular comes with a few [pipes](https://angular.io/api?type=pipe) within the framework such as DatePipe, UpperCasePipe, LowerCasePipe, CurrencyPipe etc. They are all available for use in any template.
```

Comment:
Should we add links to the API docs? E.g.:
[`DatePipe `](https://angular.io/api/common/DatePipe)

Reply:
> They are all available for use in any template.
This could be a bit misleading as the `CommonModule` from `@angular/common` needs to be imported - which is commonly imported but just to be precise.